libs/akita/src/lib/trackIdChanges.ts

```diff
@@ -0,0 +1,57 @@
+import { EntityState, EntityStore, getEntityType, getIDType, ID, QueryEntity } from '@datorama/akita';
+import { merge, MonoTypeOperatorFunction, Observable, of, Operator, Subscriber, TeardownLogic, UnaryFunction } from 'rxjs';
+import { distinctUntilChanged, filter, first, switchMap, tap } from 'rxjs/operators';
+import { OperatorFunction } from 'rxjs/src/internal/types';
+
+type getEntityStoreState<T extends EntityStore<any>> = T extends EntityStore<infer S> ? S : never;
+
+/**
+ * Track id updates of an entity and re-evaluation the query with the changed entity id.
+ * Hint: Don't place the operator after other operators in the same pipeline as those will be skipped on
```

Comment:
Also, it can be used with the selection operator. `(selectEntity(1, e => e.title))`